### PR TITLE
Fix compile error for Xcode 8 GM/Swift 3.0

### PR DESCRIPTION
**Pull request by mono0926**

### CODE REVIEW

File: CryptoSwift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -671,8 +671,10 @@
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_EMPTY_BODY = YES;
 				CLANG_WARN_ENUM_CONVERSION = YES;
+				CLANG_WARN_INFINITE_RECURSION = YES;
 				CLANG_WARN_INT_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_SUSPICIOUS_MOVE = YES;
```
Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: ? it's off by default and not used here, why change?

"Warn about suspicious uses of std::move"

Comment (mono0926) [inline CryptoSwift.xcodeproj/project.pbxproj]: Xcode 8 GM showed the warning to update to recommended setting, and I followed it.

Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: hm... I don't have such warnings, please revert, not needed.

Comment (mono0926) [inline CryptoSwift.xcodeproj/project.pbxproj]: Okay, I deleted the commit and rebased 👍

File: Tests/CryptoSwiftTests/ExtensionsTest.swift [hunk 2]
```diff
@@ -60,7 +60,9 @@ final class ExtensionsTest: XCTestCase {
         let ii:Int = 21
         XCTAssert(ii &<< 1 == ii << 1, "shift left failed")
         XCTAssert(ii &<< 8 == ii << 8, "shift left failed")
-        XCTAssert(ii &<< ((MemoryLayout<Int>.size * 8) - 1) == ii << ((MemoryLayout<Int>.size * 8) - 1), "shift left failed")
```
Comment (krzyzanowskim) [inline Tests/CryptoSwiftTests/ExtensionsTest.swift]: eh, Swift 3 compiler getting better and better

screen shot 2016-09-08 at 11 07 47

**Comment by krzyzanowskim** [comment]
Notes: Linux test failed because Swift there is no Swift 3 GM release for Linux (8/09/2016)

**Comment by krzyzanowskim** [comment]
travis-ci uses previous version of Xcode, therefore tests fails. Merge anyway.

**Pull request merged**